includes/admin/settings/class-wcmp-settings-data.php

```diff
@@ -544,18 +541,15 @@ private function get_section_carrier_postnl_delivery_options(): array
                 'help_text' => __('feature_allow_show_delivery_date_help_text', 'woocommerce-myparcel'),
             ],
             [
-                "name"      => WCMYPA_Settings::SETTING_CARRIER_DELIVERY_DAYS_WINDOW,
-                "condition" => WCMYPA_Settings::SETTING_CARRIER_ALLOW_SHOW_DELIVERY_DATE,
+                'name'      => WCMYPA_Settings::SETTING_CARRIER_DELIVERY_DAYS_WINDOW,
+                'condition' => WCMYPA_Settings::SETTING_CARRIER_ALLOW_SHOW_DELIVERY_DATE,
                 'class'      => ['wcmp__child'],
-                "label"     => __("Delivery days window", "woocommerce-myparcel"),
-                "type"      => "number",
+                'label'     => __('setting_carrier_delivery_days_window_title', 'woocommerce-myparcel'),
+                'type'      => 'number',
                 'min'       => 1,
-                "max"       => 14,
-                "default"   => self::ENABLED,
-                "help_text" => __(
-                    "Amount of days a customer can postpone a shipment. Default is 0 days with a maximum value of 14 days.",
-                    "woocommerce-myparcel"
-                ),
+                'max'       => 14,
+                'default'   => self::ENABLED,
```

Comment:
Die `self::ENABLED` kan dan toevallig wel 1 zijn, maar die constante is daar niet voor bedoeld, het "betekent" niet hetzelfde. Ik zou hier gewoon 1 schrijven.

Reply:
Scherp, deze stond er al (in huidige develop en master), heb ik niet gezien.